msg/Sensor2.msg

```diff
@@ -0,0 +1,3 @@
+float32 x
+float32 y
+float32 theta
```

Comment:
change `theta` to `psi`

Reply:
Should I change the units of psi or theta from radians to degrees?

Reply:
No, we can use radian-angle by using `psi`.